catalog/spatial/geowebcache/geowebcache-app/pom.xml

```diff
@@ -26,7 +26,7 @@
     <name>DDF :: Spatial :: GeoWebCache App</name>
 
     <properties>
-        <gwc-server.version>0.6</gwc-server.version>
+        <gwc-server.version>0.7.0-SNAPSHOT</gwc-server.version>
```

Comment:
We should only depend on released version of these 3rd party things so it doesn't cause issues when we need to do a release.  I recommend using SNAPSHOT while you're working through the upgrade but once it is working we should do the 0.7.0 release and change to that version.